codeHF/workflows.yml

```diff
@@ -50,6 +60,11 @@ workflows:
     <<: *selector_3prong
     activate: no
 
+  o2-analysis-hf-xicc-topkpipi-candidate-selector:
+    <<: *selector_2prong
```

Comment:
Inheriting from `selector_2prong` seems useless here if all the attributed are overwritten.